src/main/webapp/WEB-INF/tags/questionnaire/textQuestion.tag

```diff
@@ -0,0 +1,53 @@
+<%--
+ Copyright 2005-2014 The Kuali Foundation
+
+ Licensed under the Educational Community License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.osedu.org/licenses/ECL-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+--%>
+
+<%@ include file="/WEB-INF/jsp/kraTldHeader.jsp"%>
+
+<%@ attribute name="question" required="true" type="org.kuali.kra.questionnaire.question.Question" %>
+<%@ attribute name="answer" required="true" type="org.kuali.kra.questionnaire.answer.Answer" %>
+<%@ attribute name="questionIndex" required="true" %>
+
+<%@ attribute name="answerHeaderIndex" required="true" %>
+<%@ attribute name="bean" required="true" type="org.kuali.kra.questionnaire.QuestionnaireHelperBase" %>
+<%@ attribute name="property" required="true" %>
+<%@ attribute name="answerValidationError" required = "true" %>
+<c:set var="questionFieldName" value="${property}.answerHeaders[${answerHeaderIndex}].answers[${questionIndex}].answer" />
+<script language="javascript" src="scripts/ua_kuali_application.js"></script>

```

Comment:
You don't need to explicitly include the ua_kuali_application.js in your .tag file.
This file is set in kc-config-defaults.xml to be automatically included everywhere (in the param named kns.javascript.files)